lib/mixpanel-node.d.ts

```diff
@@ -0,0 +1,159 @@
+declare const mixpanel: mixpanel.Mixpanel;
+
+declare namespace mixpanel {
+  export type DistinctId = string;
+
+  export type ErrorCallback = (err: Error | undefined) => any;
+  export type ErrorsCallback = (errors: [Error] | undefined) => any;
+
+  type Scalar = string | number | boolean;
+
+  export interface PropertyMap {
+    [key: string]: any;
+  }
+
+  export interface NumberMap {
+    [key: string]: number;
+  }
+
+  export interface Event {
+    event: string;
+    properties: PropertyMap;
+  }
+  export interface Modifiers {
+    $ip?: string;
+    $ignore_time?: boolean;
+    $time?: string;
+    $ignore_alias?: boolean;
+  }
+
+  export interface BatchOptions {
+    max_concurrent_requests?: number;
+    max_batch_size?: number;
+  }
+
+  export interface UnionData {
+    [key: string]: Scalar | Scalar[];
+  }
+
+  interface Mixpanel {
+    init(mixpanelToken: string, config?: PropertyMap): Mixpanel;
```

Comment:
propertyMap isn't a javascript map per-say, how about calling it as `InitConfig` and defining it as `[string]: any`.

that way if InitConfig were to have named properties, it would go in there.